samples/minimal.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Can you please add a sentence here explaining what this file is?
"This file contains a sample configuration for deploying the DICOM adapters as part of a project deployed with the Data Protection Toolkit. More information is available here: https://github.com/GoogleCloudPlatform/healthcare/tree/master/deploy"

Reply:
done